### DISCUSSION TITLE
UseEmbeddedAppSettings is no more supported for Appsettings Configuration?

### DISCUSSION THREAD

**Discussion by SrinivasPam** [unoplatform/uno.extensions#621] [Q&A]
HI Team,
I have a appsettings.json file to configure application level settings. i need to configure this so that all ohter parts of application can access these settings. 

.UseEmbeddedAppSettings<App>(includeEnvironmentSettings: true)  this method that is given in Uno documentation for configuration is no more supported. looks like methods are changed. can you please help me in configuring the Appsettings for my Uno App?

below is what is provided in uno documentation for configuration
private IHost Host { get; }

public App()
{
    Host = UnoHost
        .CreateDefaultBuilder()
        **.UseEmbeddedAppSettings<App>(includeEnvironmentSettings: true)**
        .Build();
    // ........ //
}

Thanks

**Comment by nickrandolph** [comment]
Latest docs include the correct api: https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Tutorials/Configuration/HowTo-Configuration.html
